test/validate-grype-db-schema.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+import re
+import os
+import sys
+import collections
+
+pattern = re.compile(r'github.com/anchore/grype-db/pkg/db/v(?P<version>\d+)')
+
+
+def report_schema_versions_found(schema_to_locations):
+    for schema, locations in sorted(schema_to_locations.items()):
+        print("Schema: %s" % schema)
+        for location in locations:
+            print("  %s" % location)
+    print()
+
+
+def validate(schema_to_locations):
+    schema_versions_found = schema_to_locations.keys()
+    try:
+        for x in schema_versions_found:
+            int(x)
+    except Exception:
```

Comment:
(not a Python expert by any means 😄 ) Is catching a bare `Exception` something that we should try to avoid?

Reply:
yup 👍 I'll scope this a bit narrower